custom_components/tahoma/config_flow.py

```diff
@@ -12,12 +11,8 @@
 from homeassistant.core import callback
 from homeassistant.helpers import config_validation as cv
 
-from .const import (
-    CONF_UPDATE_INTERVAL,
-    DEFAULT_UPDATE_INTERVAL,
-    DOMAIN,
-    MIN_UPDATE_INTERVAL,
-)
+from .const import CONF_UPDATE_INTERVAL, DEFAULT_UPDATE_INTERVAL, MIN_UPDATE_INTERVAL
+from .const import DOMAIN  # pylint: disable=unused-import
```

Comment:
Pylint don't like this?

Reply:
Nope... I am not 100% sure why, but every other component is disabling this warning for this line indeed.